frontend/assets/stylesheets/components/_action.scss

```diff
@@ -125,11 +125,10 @@
 .action--booking {
     color: $white;
     background-color: $black;
-    border-color: $black;
+    border: 0 none;
```

Comment:
Yeah, the former. `border: none` or `border: 0` would be enough on their own, right?